virtManager/config.py

```diff
@@ -212,6 +212,17 @@ def embeddable_graphics(self):
         ret = ["vnc", "spice"]
         return ret
 
+    def get_perfered_graphics_index(self, gdevs):
+        try:
+            for idx, dev in enumerate(gdevs):
+                if dev.type in self.embeddable_graphics():
+                    return idx
+            if len(gdevs) > 0:
+                return 0
+            return None
+        except:
```

Comment:
This except block should be removed. if we screw up the basic code inside, we want to know about it

Reply:
This function was removed.